Library/Formula/hashcat.rb

```diff
@@ -0,0 +1,34 @@
+class Hashcat < Formula
+  desc "Advanced CPU-based password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://codeload.github.com/hashcat/hashcat/tar.gz/2.00"
```

Comment:
Since the `url` and `sha256` are for the stable version, it's easier to read the formula if they're in the `stable do` block as well.